pygmt/src/histogram.py

```diff
@@ -54,7 +54,9 @@ def histogram(self, table, **kwargs):
         Pass in either a file name to an ASCII data table, a Python list, a 2D
         {table-classes}.
     {J}
+        *Required if this is the first plot command.*
     {R}
+        *Required if this is the first plot command.*
```

Comment:
`-J` and `-R` are not required for `histogram`.

This CLI command works well without `-J` and `-R`:
```
gmt histogram @v3206_06.txt -F -T250 -W0.25p -B -pdf plot
```

The comment in the [pshistogram.c source code](https://github.com/GenericMappingTools/gmt/blob/3ebaae9ad5385309e5ec89f80fbce0cad6a30c7c/src/pshistogram.c#L35) explains why `THIS_MODULE_NEEDS` is `JR`, but `-J` and `-R` are not required:

> Note: The NEEDS must be JR.  Although pshistogram can create a region from data, it
> does so indirectly by building the histogram and setting the ymin/ymax that way, NOT by
> reading y data (which is what a NEEDS of d or r would do).

Reply:
fixed in f8f9cf8